apps/custom-app/values.yaml

```diff
@@ -0,0 +1,54 @@
+appname: customapp
+project:
+  name: project
+  slug: project-slug
+
+permission: public
+
+global:
+  protocol: TCP'
```

Comment:
Overall looks good. Great job. Just a question on this. Is this a typo or is there a reason that there is a single quote after the protocol?

Reply:
No, typo! Great find!